### DISCUSSION TITLE
White screen with IOS

### DISCUSSION THREAD

**Discussion by jocelynkerbourch** [ionic-team/capacitor#3559] [Q&A]
Hello,
I wanted to test capacitor for IOS with our PWA which is available here https://employee.yuco.app/. But I have a blank page.
My front is on React and I am using Yarn.

here is my capacitor.config.json
```
{
  "appId": "co.yuco.app",
  "appName": "Yuco",
  "bundledWebRuntime": false,
  "npmClient": "yarn",
  "webDir": "public",
  "plugins": {
    "SplashScreen": {
      "launchShowDuration": 0
    }
  },
  "cordova": {}
}
```
and then executed
```
npm run build   
npx cap copy  
npx cap open ios 
```
Thanks for your help

**Comment by jcesarmobile** [comment]
White screens are usually caused by javascript errors

you can watch this video about how to debug iOS apps
https://www.youtube.com/watch?v=7XgrrO1h7Wo

**Reply by jc0d3d3v** [reply]
White screen issue is tricky..It happens in Android as well without Javascript errors.  https://github.com/ionic-team/capacitor/discussions/3368

**Reply by jcesarmobile** [reply]
that's not the same thing
